db/blip_sync_context.go

```diff
@@ -274,8 +277,13 @@ func (bsc *BlipSyncContext) handleChangesResponse(sender *blip.Sender, response
 			revSendTimeLatency += time.Since(changesResponseReceived).Nanoseconds()
 			revSendCount++
 
-			if bsc.preSendRevisionResponseCallback != nil {
-				bsc.preSendRevisionResponseCallback(seq.String())
+			if bsc.sgr2PushAddExpectedSeqCallback != nil {
```

Comment:
Similar to comment elsewhere - what do you think about changing this API to accept []string and batching up the changes so that it's called once per handleChangesResponse invocation?